test/src/main/java/org/apache/accumulo/test/upgrade/UpgradeIT.java

```diff
@@ -139,14 +167,30 @@ public void testUpgradeManagerFailure() throws Exception {
     ServerContext ctx = getCluster().getServerContext();
     UpgradeUtilIT.downgradePersistentVersion(ctx);
 
-    List<String> args = new ArrayList<>();
-    args.add("--props");
-    args.add(getCluster().getAccumuloPropertiesPath());
-    try (TestManager mgr = new TestManager(args.toArray(new String[0]))) {
-      IllegalStateException ise = assertThrows(IllegalStateException.class, () -> mgr.runServer());
-      assertTrue(ise.getMessage().equals("Upgrade not started, " + Constants.ZUPGRADE_PROGRESS
-          + " node does not exist. Did you run 'accumulo upgrade --start'?"));
-    }
+    final AtomicReference<Throwable> err = new AtomicReference<>();
+    Thread testMgrThread = new Thread() {
+      @Override
+      public void run() {
+        try {
+          List<String> args = new ArrayList<>();
+          args.add("--props");
+          args.add(getCluster().getAccumuloPropertiesPath());
+          TestManager mgr = new TestManager(args.toArray(new String[0]));
+          mgr.runServer();
```

Comment:
What is the benefit of calling this code in another thread?

Reply:
Made a comment elsewhere about close and try w/ resources.  Not sure why, but it seems like not calling close in try w/ resource and the change to a different server context avoid halting the test process.

Reply:
> I changed TestManager constructor back to the following locally super(new ConfigOpts(), (conf) -> getServerContext(), args) and ran these updated test and it still fails

I have reproduced this locally as well.

Reply:
I suspect the thread code could be removed, simplifying the test.  Assuming the test thread called something that halted the process.  If there is an uncaught exception from the test, junit code should catch that and handle in its normal way.